packages/blade/src/components/Chip/index.ts

```diff
@@ -0,0 +1,2 @@
+export * from './Chip';
```

Comment:
why export everything? export only required things? like type and the component? otherwise all internal components will be exported.

Reply:
Currently, we are exporting only Chip and ChipProps here. but I do see the point here. We should update all components to export only the required types & components. Maybe we can add an eslint rule for this. Currently, there are many components with `export * from './component';` exports.